lib/aggregation/accumulator/src/index.js

```diff
@@ -351,9 +351,13 @@ const runCLI = () => {
 
   // Create the app, replay any failed events, and listen
   const app = accumulator();
-  dataflow.replay(app.reducer, 0, () => {
-    app.listen();
-  });
+  app.listen();
+  if (cluster.isWorker()) {
```

Comment:
We can remove this check since:
1. As @KRuelY noticed, we already have a check in dataflow that prevents `replay` running more than once
2. We should be running `replay` on master since workers can be > 1